web/scenes/Portal/Teams/TeamId/Team/page/Members/List/EditRoleDialog/index.tsx

```diff
@@ -46,7 +46,7 @@ const schema = yup.object({
 type FormValues = yup.InferType<typeof schema>;
 
 export const EditRoleDialog = (props: {
```

Comment:
@michaelfurutin May I ask you to adjust order of roles in the list? 
Now it's 
- Admin
- Member
- Owner. 

Can we sort it by permissions? As it is displayed in the permissions list dialog. 
- Owner
- Admin
- Member

Reply:
fixed